server/src/test/java/io/spine/server/aggregate/AggregatePartRepositoryLookupTest.java

```diff
@@ -72,27 +71,27 @@ public void find_a_repository() {
     }
 
     @Test
-    public void throw_if_repository_is_not_AggregatePartRepository() {
+    @DisplayName("throw if repository not found")
+    void throwOnRepositoryNotFound() {
         AggregatePartRepositoryLookup lookup =
-                createLookup(boundedContext, TaskId.class, Task.class);
-        thrown.expect(IllegalStateException.class);
-        lookup.find();
+                createLookup(boundedContext, Timestamp.class, StringValue.class);
+        assertThrows(IllegalStateException.class, lookup::find);
     }
 
     @Test
-    public void throw_if_repository_not_found() {
+    @DisplayName("throw if found repository is not AggregatePartRepository")
+    void throwOnWrongRepositoryTypeFound() {
         AggregatePartRepositoryLookup lookup =
-                createLookup(boundedContext, Timestamp.class, StringValue.class);
-        thrown.expect(IllegalStateException.class);
-        lookup.find();
+                createLookup(boundedContext, TaskId.class, Task.class);
+        assertThrows(IllegalStateException.class, lookup::find);
     }
 
     @Test
-    public void throw_if_id_class_does_not_match() {
+    @DisplayName("throw if found repository id class does not match root id class")
```

Comment:
Similar to the one above. Throw what?

`if the ID class of the repo found does not match the ID class of its root`.